src/openssl.cr

```diff
@@ -1,6 +1,6 @@
 require "./openssl/lib_ssl"
 
-# # OpenSSL Integration
+# ## Integration
```

Comment:
Why remove the `OpenSSL` here? It might be a bit redundant, but I think it's anyway more clear in this case to leave `OpenSSL Integration`